app/static/js/createEvents.js

```diff
@@ -8,14 +9,15 @@ function updateDate(obj) {
   var newMonth = dateToChange.getMonth();
   var newYear = dateToChange.getFullYear();
   var newDay = dateToChange.getDate();
-  if(obj.id == "startDatePicker") {
-    $("#endDatePicker").datepicker({minDate: new Date(  newYear, newMonth, newDay)});
-    $("#endDatePicker").datepicker( "option", "minDate", new Date(  newYear, newMonth, newDay));
+  if(obj.className.includes("startDatePicker")) {
+    
+    $("#endDatePicker-"+$(obj).data("page-location")).datepicker({minDate: new Date(  newYear, newMonth, newDay)});
+    $("#endDatePicker-"+$(obj).data("page-location")).datepicker( "option", "minDate", new Date(  newYear, newMonth, newDay));
   }
 
-  if (obj.id == "endDatePicker") {
-    $("#startDatePicker").datepicker({maxDate: new Date(  newYear, newMonth, newDay)});
-    $("#startDatePicker").datepicker("option", "maxDate", new Date(  newYear, newMonth, newDay));
+  if (obj.className.includes("endDatePicker")) {
+    $("#startDatePicker-"+$(obj).data("page-location")).datepicker({maxDate: new Date(  newYear, newMonth, newDay)});
```

Comment:
Same on these lines with the two spaces after Date(

Reply:
Fixed